wordpress-mobile/WordPress-Android
# Quick Start: Update snackbars style on non my site screens

Fixes #15462

This PR 
- Displays `Quick Start` prompt on other screens (`Page List`, `Reader` and `Publicise`) using the snackbar sequencer. enabling us to remove the functions to [show and hide the `Quick Start` snackbar](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java#L1565-L1578) from the `WPMainActivity`.
- Updates non-themed style on these snackbars using the [themed style used for `My Site - Quick Start`](https://github.com/wordpress-mobile/WordPress-Android/blob/92dfe6dcb9b6c3f8bd886f95052a455ff0ebfd68/WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteFragment.kt#L230).

To test:

1. Let `Quick Start` card display after creating a new site.
2. Perform below `Quick Start` steps (involving 2-steps, with the second snackbar displaying on the non `My Site` screen )
    Quick Start Step| Target Screen 
    -----------------|---------------
    `Customise your site -> Edit your homepage ` | `Page List`
    `Grow your audience -> Social sharing` | `Publicise`
    `Grow your audience -> Follow other sites`| `Reader`
3. Notice that the snackbar is displayed using the themed style used for prompts on the `My Site` screen.

#### Screenshots for `Quick Start` snackbar on `Page List`
(Same style is applied on the other target screens)

**Light Mode**

Before | After
-------|-------
<img width="442" alt="orig_light" src="https://user-images.githubusercontent.com/1405144/143565538-d3c773af-09d4-4b96-ac6e-99b3bc3e8901.png"> | <img width="441" alt="new_light" src="https://user-images.githubusercontent.com/1405144/143565130-568d5e9c-e9f3-4049-81b4-f4ce83278b8f.png">


**Dark Mode**

Before | After
-------|-------
<img width="441" alt="orig_dark" src="https://user-images.githubusercontent.com/1405144/143565178-6f937732-200b-4ea7-99da-db2d468e9d11.png"> | <img width="439" alt="new_dark" src="https://user-images.githubusercontent.com/1405144/143565196-d2b5d1fe-3f84-44c8-930a-473cd1d8f2f3.png">

**Merge Instructions**
 
1. Make sure design review is done.
2. Remove `Needs Design Review `, `Not Ready for Merge` labels.
3. Merge the PR.

The PR targets `18.9` instead of the `18.8` milestone (even though it can be merged with `18.8`) to avoid any rush for the review.

## Regression Notes
1. Potential unintended areas of impact
N/A (minimal changes done only to update the style).

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
